asset/css/compat.less

```diff
@@ -25,6 +25,15 @@ form.icinga-form {
   }
 }
 
+.icinga-controls {
+  .required-hint {
+    font-size: 1.2em;
```

Comment:
Please don't change the font size. This results in a higher than normal line height and offsets most of the form elements.